app/scripts/modules/amazon/src/pipeline/stages/deployCloudFormation/evaluateCloudFormationChangeSetExecutionMarkerIcon.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { IExecutionMarkerIconProps } from 'core/pipeline/config/stages/common/ExecutionMarkerIcon';
+
+export class EvaluateCloudFormationChangeSetExecutionMarkerIcon extends React.Component<IExecutionMarkerIconProps> {
+  constructor(props: IExecutionMarkerIconProps) {
+    super(props);
+  }
+
+  public render() {
+    if (
+      this.props.stage.isRunning &&
+      this.props.stage.stages[0].context.changeSetContainsReplacement &&
+      this.props.stage.stages[0].context.actionOnReplacement === 'ask'
+    ) {
+      require('./evaluateCloudFormationChangeSetExecutionDetailsAsk.less');
```

Comment:
This is an anti-pattern. We typically everything from other files at the top.

Reply:
`import './evaluateCloudFormationChangeSetExecutionDetailsAsk.less'`

Reply:
Not longer needed with https://github.com/spinnaker/deck/pull/7671/commits/b088c914c75463bf966663c866624f61473cb9fa